frontend/resourceadm/pages/MigrationPage/MigrationPage.tsx

```diff
@@ -1,109 +1,222 @@
+                tjenesten har lite eller ingen trafikk. For eksempel midt på natten.
+              </Paragraph>
+              <div className={classes.datePickers}>
+                <div className={classes.datePickerWrapper}>
+                  <TextField
+                    type='date'
+                    value={migrationDate}
+                    onChange={(e) => setMigrationDate(e.target.value)}
+                    label='Migreringsdato'
+                  />
+                </div>
+                <div className={classes.datePickerWrapper}>
+                  <TextField
+                    type='time'
+                    value={migrationTime}
+                    onChange={(e) => setMigrationTime(e.target.value)}
+                    label='Klokkeslett'
+                  />
+                </div>
+              </div>
+              <div className={classes.numDelegations}>
+                <Label size='medium' spacing>
+                  Antall delegeringer i Altinn 2 og Altinn 3
+                </Label>
+                <Button
+                  onClick={() => {
+                    // TODO - replace with API call
+                    setNumDelegationsA2(1000);
+                    setNumDelegationsA3(1000);
+                  }}
+                  className={classes.button}
+                >
+                  Hent antall delegeringer
+                </Button>
+                {numDelegationsA2 && numDelegationsA3 && (
+                  <div className={classes.delegations}>
+                    <Paragraph size='small'>
+                      Altinn 2: <strong>{numDelegationsA2}</strong> delegeringer
+                    </Paragraph>
+                    <Paragraph size='small'>
+                      Altinn 3: <strong>{numDelegationsA3}</strong> delegeringer
+                    </Paragraph>
+                  </div>
+                )}
+              </div>
+              <Label size='medium' spacing>
+                Fullfør migrering
+              </Label>
+              <Paragraph size='small'>
+                For at brukere med eksisterende tilganger til tjenesten i Altinn 2 skal videreføre
+                sine tilganger for ny tjeneste migrert til Altinn 2 må delegeringene migreres til
+                Altinn 3. Les detaljer om prosessen i vår dokumentasjon.
+              </Paragraph>
+              <div className={classes.buttonWrapper}>
+                <Button
+                  disabled={
+                    !(
+                      validateResourceData.status === 200 &&
+                      validatePolicyData?.status === 200 &&
+                      deployOK
+                    )
+                  }
+                  onClick={() => {}}
+                  className={classes.button}
+                >
+                  Migrer delegeringer
+                </Button>
+                <Button disabled onClick={() => {}} className={classes.button}>
```

Comment:
Consider using aria-disabled instead of disabled to support a better experience for those that navigate using tab and screen readers. This is the recommended way when using [design system buttons](https://digdir.github.io/designsystem/iframe.html?viewMode=docs&id=kjernekomponenter-button--docs#bruk-av-spinner-til-%C3%A5-lage-en-knapp-som-laster)